Migrate Amount step of Send to the new design - Closes #760

colado opened the pull request:
<!---
Hints for a successful PR:
1. Please open an issue before starting to fix a bug or implementing a feature, in order to make sure your changes in aligned with the project goals. This way, you'll also find out if anyone else is working on a similar change.
2. Please do open a PR introducing big chunks of changes in a lot of files. instead, please consider breaking the issue into multiple smaller issues.
3. Please fill out the template below for ease of review.
-->

# What was the bug or feature?
Described in #760 

### How did I fix it?
Changed style of Amount component and any related components.

### How to test it?
Navigate to Send and select the address to send tokens to.


# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes

reyraa commented:
Looks good. The only issue I can see is that the BTC processing speed and fee are missing. Pleas take a look at the [screens in InVision](https://projects.invisionapp.com/d/main#/console/17093670/358879594/build).

reyraa commented:
Additionally, although not in the design files, I'd use a bigger font for the estimation sign `~`, in order to not look like minus `-` sign.

colado commented:
@reyraa I've added the changes to the processing fee component and also changed the size of the estimation sign. Let me know if there's anything else that needs to be changed.
